numpyro/infer/hmc.py

```diff
@@ -199,7 +199,7 @@ def init_kernel(
         regularize_mass_matrix=True,
         model_args=(),
         model_kwargs=None,
-        rng_key=random.PRNGKey(0),
+        rng_key=None,
```

Comment:
This change is used to avoid invoking jax operations when rendering the docs. This makes doc rendering faster.

Reply:
I see.